src/python/pants/help/help_printer.py

```diff
@@ -127,6 +127,7 @@ def _help_table(
                 self._all_help_info.name_to_api_type_info.keys(), self._print_api_type_help
             ),
             **_help_table(self._all_help_info.name_to_rule_info.keys(), self._print_rule_help),
+            **_help_table(self._all_help_info.env_var_to_help_info, self._print_env_var_help),
```

Comment:
```suggestion
            **_help_table(self._all_help_info.env_var_to_help_info.keys(), self._print_env_var_help),
```

For consistency with the lines above it, and also because it's a reminder to the reader that `_help_table` expects an iterable, not a dict.

Reply:
in general I felt the explicit `.keys()` calls were unnecessary (I'd have omitted them if writing from scratch), but it's good to maintain consistency and it's not a big deal either way - fixed this